static/templates/stream_settings/stream_types.hbs

```diff
@@ -1,4 +1,4 @@
-<div class="grey-box">
+<div class="grey-box new-style">
```

Comment:
@ganpa3, working on this area in relation to the follow up #20222, I'm wondering do we really need this "new-style" class?
It seems to only be used to apply background-color and order-color on the dropdown and even then, `body.color-scheme-automatic select` or `body.night-mode select` will handle the background-color and border-color without it.

Reply:
@YashRE42 Change the dropdown value of `Message retention period` to `Retain for N days after posting` and remove the `new-style` class to see the effect.

![image](https://user-images.githubusercontent.com/58626718/141502814-b32bac68-aee4-4d5e-b63b-6800806536f0.png)

Reply:
Thanks, I can work on that.